src/main/webapp/script.js

```diff
@@ -12,58 +12,314 @@
+        textRow1.appendChild(row1Col1);
+        textRow1.appendChild(row1Col2);
+        textRow1.appendChild(row1Col3);
+        text.appendChild(textRow1);
+        const hr = document.createElement('hr');
+        taskElement.appendChild(usPfp);
+        taskElement.appendChild(text);
+        fulltask.appendChild(taskElement);
+        fulltask.appendChild(hr);
+        return fulltask;
+    } else {
+        const fulltask = document.createElement('div');
+        fulltask.setAttribute("id", 'task');
+        //crear bloque de info users
+        const taskElement = document.createElement('div');
+        taskElement.className = 'users-drawer  rounded-1';
+        //dentro de bloque, poner imagen de perfil
+        const usPfp = document.createElement("i");
+        usPfp.className = "bi bi-person-circle uspfp";
+
+        //dentro de bloque, crear formato del texto de info
+        const text = document.createElement("div");
+        text.className = "text";
+
+        //dentro de texto, crear primera fila
+        const textRow1 = document.createElement("div");
+        textRow1.className = "row m-0";
+        //dentro de primera fila, crear columna 1
+        const row1Col1 = document.createElement("div");
+        row1Col1.className = "col-2 ps-1";
+        //dentro de columna 1, poner titulo/user name
+        const r1c1h = document.createElement("h6");
+        r1c1h.innerText = username;
+
+        //dentro de primera fila, crear columna 2
+        const row1Col2 = document.createElement("div");
+        row1Col2.className = "col-1";
+        //dentro de columna 2, poner icon de flecha
+        const r1c2h = document.createElement("i");
+        r1c2h.className = "bi bi-arrow-right flech";
+
+        //dentro de primera fila, crear columna 3
+        const row1Col3 = document.createElement("div");
+        row1Col3.className = "col-3";
+        //dentro de columna 3, poner titulo task
+        const r1c3h = document.createElement("h6");
+        r1c3h.innerText = task.title;
+
+
+        //dentro de primera fila, crear columna 5
+        const row1Col4 = document.createElement("div");
+        row1Col4.className = "col-4 b ps-2";
+        //dentro de columna 3, poner titulo task
+        const r1c4h = document.createElement("h6");
+        r1c4h.innerText = "Time Goal: " + task.time + " min";
+
+        //dentro de primera fila, crear columna 5
+        const row1Col5 = document.createElement("div");
+        row1Col5.className = "col-1 text-center";
+        //dentro de columna 3, poner titulo task
+        const r1c5h = document.createElement("i");
+        r1c5h.className = "bi bi-dash-lg";
+
+        //dentro de primera fila, crear columna 4
+        const row1Col6 = document.createElement("div");
+        row1Col6.className = "col-1";
+        //dentro de columna 3, poner titulo task
+        const r1c6h = document.createElement("h6");
+        r1c6h.innerText = format_time(task.timeCreated);
+
+
+
```

Comment:
Also, could we display either the `task.end` if the task is completed or the time when the time goal would be met if the task is not completed?  For the time goal calculation, I did some prototyping in the console and I think the following would work.

Example task that has not been completed yet:

```javascript
{
desc: "https://github.com/avrittrohwer/software-product-sprint-2022/pull/5/files"
end: 1657993938925
start: 1657993938925
time: "30"
title: "Review PR"
}
```

```javascript
if (task.start != task.end) {
  // Display task end time
  someElement.innerText = time_format(task.end)
} else {
  // Display when time goal will be met
  d = new Date(task.start)
  d.setMinutes(d.getMinutes() + parseInt(task.time))
  someElement.innerText = time_format(d)
}
```

Reply:
Done.... 
![task finished](https://user-images.githubusercontent.com/105701551/179373330-57271413-672d-4875-97c0-09a75cbcc602.png)